### DISCUSSION TITLE
Ignore transparency on images

### DISCUSSION THREAD

**Discussion by TT-ReBORN** [reupen/columns_ui#818] [Ideas] [answered]
Hi @reupen,

I dunno if it is technical possible to achieve this but it would be cool to have an additional check mark option to ignore applying transparency on images when using: 
`Preferences` > `Display` > `Columns UI` > `Miscellaneous` > `Use transparency, opacity`.
I think when using reasonable overall opacity values ( e.g 204 -> 80% ), large album art and thumbnail images would look better
if no transparency would be applied and keep them on 100% opacity.

If it is possible I would like to request this. I think more people would use then the Columns UI transparency feature.

Thanks

-TT

**Comment by reupen** [comment, marked as answer]
Hi,

It's not easily possible. The transparency is essentially an attribute that Windows allows setting on the main window, and it affects all its contents.

Windows 8 and newer do let you make child windows (e.g. the playlist view) transparent, but that's the opposite of what you are asking, and a transparent child window would take on the background of its parent window (usually a splitter panel) rather than whatever's underneath the main window.

**Reply by TT-ReBORN** [reply]
I had the thought in mind that maybe you could somehow make a rule to exclude this for `GdiBitmap` objects.
Disappointing that Windows is so unflexible :-(

**Reply by TT-ReBORN** [reply]
Btw, UI Hacks does also have a transparency feature:
`Preferences`> `Display` > `Main Window` > `Aero effects` > `Alpha transparency`
It is crap ( bugged anti-aliasing ) because it removes ( 0% opacity, feels pretty cheap and not good ) all backgrounds with its colors but does not apply it on `GdiBitmap` objects:
https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/1#discussioncomment-7415637

So I was in hope this could be also done with Columns UI...

**Reply by reupen** [reply]
Not sure exactly what it's doing, but remember UI Hacks is a hack. It's probably overriding the rendering of all panels which is not something that another component should be doing.
